test/Microsoft.AspNetCore.Mvc.Core.Test/ApiConventionTypeAttributeTest.cs

```diff
@@ -2,6 +2,7 @@
 // Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
 
 using System;
+using System.Linq;
```

Comment:
How do changes to this class relate to #6828?

Reply:
The error message changed. Refactoring it so we don't have to change it any time we add a new attribute that we allow on a convention.

Reply:
Ah, IOW the real change was buried 😈 
:ok: